<?xml version="1.0" encoding="utf-8"?>
<launch>

   <node name="rviz" pkg="rviz" type="rviz" output="screen"
    args="-d $(find curve_gen)/config/curve_gen.rviz"> 
  </node>
 
  <node pkg="curve_gen" type="curve_gen" name="curve_gen_node" output="screen">
    <rosparam file="$(find curve_gen)/config/curve_gen.yaml" command="load" />
  </node>

</launch>
